sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -875,6 +912,30 @@ private void setDefaultTransmissionTarget(String transmissionTargetToken) {
         }
     }
 
+    /**
+     * Set trigger interval. The interval should be between 3 seconds and 1 day.
```

Comment:
This doc is saying "trigger interval" which isn't consistent with the member variable name. I think we can keep it if the variable name is changed.

Reply:
I think the specification of the new API is transmission interval, use the new name here.